schema/attribute/legal-jurisdiction.json

```diff
@@ -1,267 +1,520 @@
+		"Kenya",
+		"Kiribati",
+		"Kosovo",
+		"Kuwait",
+		"Kyrgyzstan",
+		"Laos",
+		"Latvia",
+		"Lebanon",
+		"Lesotho",
+		"Liberia",
+		"Liechtenstein",
+		"Lithuania",
+		"Luxembourg",
+		"Macedonia",
+		"Madagascar",
+		"Malta",
+		"Malawi",
+		"Malaysia",
+		"Maldives",
+		"Mali",
+		"Marshall Islands",
+		"Mauritania",
+		"Mauritius",
+		"Mexico",
+		"Micronesia, Federated States of",
+		"Moldova",
+		"Monaco",
+		"Mongolia",
+		"Montenegro",
+		"Montserrat",
+		"Morocco",
+		"Mozambique",
+		"Myanmar",
+		"Namibia",
+		"Nauru",
+		"Nepal",
+		"Netherlands",
+		"Bonaire, Sint Eustatius and Saba",
+		"New Zealand",
+		"Nicaragua",
+		"Niger",
+		"Nigeria",
+		"Niue",
+		"Norway",
+		"Oman",
+		"Pakistan",
+		"Palau",
+		"Palestinian territories",
+		"Panama",
+		"Papua New Guinea",
+		"Paraguay",
+		"Peru",
+		"Philippines",
+		"Poland",
+		"Portugal",
+		"Qatar",
+		"Romania",
+		"Russian Federation",
+		"Rwanda",
+		"Saint Kitts and Nevis",
+		"Saint Lucia",
+		"Saint Vincent and the Grenadines",
+		"Samoa",
+		"San Marino",
+		"São Tomé and Príncipe",
+		"Saudi Arabia",
+		"Senegal",
+		"Serbia",
+		"Seychelles",
+		"Sierra Leone",
+		"Singapore",
+		"Slovakia",
+		"Slovenia",
+		"Solomon Islands",
+		"South Africa",
+		"South Korea",
+		"South Sudan",
+		"Spain",
+		"Sri Lanka",
+		"Sint Maarten",
+		"Sudan",
+		"Suriname",
+		"Swaziland",
+		"Sweden",
+		"Switzerland",
+		"Tajikistan",
+		"Tanzania",
+		"Thailand",
+		"Timor-Leste",
+		"Tonga",
+		"Togo",
+		"Tunisia",
+		"Turkey",
+		"Turks and Caicos Islands",
+		"Trinidad & Tobago",
+		"Uganda",
+		"Ukraine",
+		"United Arab Emirates",
+		"United Kingdom",
+		"United States - Alabama",
+		"United States - Alaska",
+		"United States - Arizona",
+		"United States - Arkansas",
+		"United States - California",
+		"United States - Colorado",
+		"United States - Connecticut",
+		"United States - Delaware",
+		"United States - Florida",
+		"United States - Georgia",
+		"United States - Hawaii",
+		"United States - Idaho",
+		"United States - Illinois",
+		"United States - Indiana",
+		"United States - Iowa",
+		"United States - Kansas",
+		"United States - Kentucky",
+		"United States - Louisiana",
+		"United States - Maine",
+		"United States - Maryland",
+		"United States - Massachusetts",
+		"United States - Michigan",
+		"United States - Minnesota",
+		"United States - Mississippi",
+		"United States - Missouri",
+		"United States - Montana",
+		"United States - Nebraska",
+		"United States - Nevada",
+		"United States - New Hampshire",
+		"United States - New Jersey",
+		"United States - New Mexico",
+		"United States - New York",
+		"United States - North Carolina",
+		"United States - North Dakota",
+		"United States - Ohio",
+		"United States - Oklahoma",
+		"United States - Oregon",
+		"United States - Pennsylvania",
+		"United States - Rhode Island",
+		"United States - South Carolina",
+		"United States - South Dakota",
+		"United States - Tennessee",
+		"United States - Utah",
```

Comment:
@sk91 it looks like you've left out `United States - Texas`. Was this on purpose?

Reply:
@Chaviland7 thous are the jurisdictions we had from FT.

@andregoncalves did we miss one?

Reply:
I think we missed that. It's on the GLEIF legal jurisdictions codes, so it's a valid jurisdiction.
BTW I'm waiting for Marc's reply that list.

Reply:
fixed